client/stylesheets/_responsive.scss

```diff
@@ -170,4 +259,10 @@
     .helpImgUl > li:nth-child(2) {
     	display: none !important;
     }
+    .profileCardLi {
+    	width: 90% !important;
+    }
+    .searchResultsCardLi {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `searchResultsCardLi` should be written in lowercase with hyphens